nb2/bot/slack_bot.py

```diff
@@ -220,6 +226,26 @@ def remind(self, nostalgia_user_target: str, slack_user_targets: List[str]):
         )
         return self.Result(ok=True, message=message)
 
+    def random(self):
+        """
+        Recall a random quote from NB's memory.
+
+        Returns:
+            A Result namedtuple.
+        """
+        person = get_random_person()
+
+        if person is None:
+            return self.Result(ok=True, message="No memories to remember")
+
+        real_name, quote = self._random_quote(person)
+
+        if quote is None:
+            return self.Result(ok=True, message="No memories to remember")
```

Comment:
Is is possible to get to a Person with no quotes?

Reply:
Technically yes, but it also "shouldn't happen" since we'll typically be adding users only when we're also adding a quote for them.